src/microbe_stage/Microbe.Contact.cs

```diff
@@ -1439,6 +1439,17 @@ private void OnContactBegin(int bodyID, Node body, int bodyShape, int localShape
                     return;
 
                 var target = otherIsPilus ? thisMicrobe : touchedMicrobe;
+                var prey = otherIsPilus ? touchedMicrobe : thisMicrobe;
```

Comment:
The target variable is already the prey. i.e. it is the cell that didn't have a pilus and is hit.